os/kernel/init/os_start.c

```diff
@@ -419,6 +419,10 @@ void os_start(void)
 	}
 #endif
 
+#ifdef CONFIG_APP_BINARY_SEPARATION
```

Comment:
Better to keep inside Initialize the memory manager  block

Reply:
Even though this is related to memory management, I dont see any dependency on any of these configs. 
#if defined(MM_KERNEL_USRHEAP_INIT) || defined(CONFIG_MM_KERNEL_HEAP) || defined(CONFIG_MM_PGALLOC)
So, I have placed this initialization just after other memory initializations.

I added a brief description.